src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd

```diff
@@ -0,0 +1,191 @@
+@echo off
+setlocal
+
+set Id=%1
+set IdMaster=Master
+set WcfTestDir=c:\WCFTest
+set RepoHome=c:\git
+set CurrentRepo=%RepoHome%\wcf%Id%
+set MasterRepo=%RepoHome%\wcf%IdMaster%
+set PRServiceName=PRService%Id%
+set WcfServiceName=WcfService%Id%
+set PRServiceMaster=PRService%IdMaster%
+set WcfServiceMaster=WcfService%IdMaster%
+set LogFile="%~dp0%~n0.log"
+set CertExpirationInDays=99
+set ExitCode=0
+set cmd=
+
+if '%1'=='' goto :Usage
+if '%1'=='/?' goto :Usage
+if '%1'=='-?' goto :Usage
+if '%1'=='/help' goto :Usage
+if '%1'=='-help' goto :Usage
+
+:: Make sure IIS appcmd.exe is accessible
+set appcmd=%windir%\System32\inetsrv\appcmd.exe
+if NOT EXIST %appcmd% (
+    echo Could not find: %appcmd%.
+    echo Please make sure IIS is installed.
+    goto :Failure
+)
+
+:: Clean up any existing setup
+echo Deleting WCF repo at %CurrentRepo% if exists and associated application pools and services hosted on IIS...
+if exist %CurrentRepo% rmdir /s /q %CurrentRepo%
+if exist %WcfTestDir% if /I '%MasterRepo%'=='%CurrentRepo%' rmdir /s /q %WcfTestDir%
+%appcmd% delete app "Default Web Site/%PRServiceName%" >nul
```

Comment:
Nit and not blocking: a failed clean *may* not mean much for the problems you expect.  The reason I recommend logging is for the failures you didn't.  I still think it would be a good idea for this script to not suppress stdout and let the person who ran it decide what was important.  Your call.